tests/test_core/test_alignment.py

```diff
@@ -2469,6 +2479,31 @@ def test_slice_align_info(self):
 class AlignmentTests(AlignmentBaseTests, TestCase):
     Class = Alignment
 
+    def test_sliced_deepcopy(self):
+        """correctly deep copy aligned objects in an alignment
+
```

Comment:
test method docstrings should be a single short line because it get's displayed in the terminal on failures. Make the other remarks a comment. But that said, I don't think force_same data should be part of the interface, so those test modifications should be reversed.